### DISCUSSION TITLE
Usage of Budibase

### DISCUSSION THREAD

**Discussion by mothtu** [Budibase/budibase#13813] [Help] [answered]
I'm not sure if this is the right place to ask this, but is it allowed to use Budibase as a visual interface for a database that my school has for all the written projects that the students have made? I'm working on a project for my school to create a database for it and this is the best solution i've found so far.

**Comment by mikesealey** [comment, marked as answer]
Hi @mothtu

This sounds like a great use of Budibase! 

You can find our documentation [here](https://docs.budibase.com/docs/what-is-budibase), and we also have a helpful and supportive community in our [Discord Server](https://discord.com/invite/ZepTmGbtfF), where you could show off your skills or get ideas and help from other Budibase users.
